test/arrayChanges.spec.js

```diff
@@ -354,5 +394,21 @@ describe('array-changes-async', function () {
             ]);
         });
     }
+
+    it('produces a valid plan', function () {
+        var g = generators(42);
```

Comment:
On my puny work machine I had to increase the timeout to 4000 milliseconds before this test passed.